sensor_msgs/Image image

float32 gps_lati
float32 gps_longit
string target_color
string target_shape
string symbol
string symbol_color
string orientation
